blocks/core/ff_module/ff_module-task-event/src/TaskEventContainer.rt

```diff
@@ -0,0 +1,6 @@
+<div
```

Comment:
Base feels a bit more appropriate than wrapper, so I'd be happy with that. (It's not really exposed anywhere outside of the pattern library though, so from Firefly's point of view it doesn't matter too much.)